changelog/2288.changed.md

```diff
@@ -0,0 +1 @@
+Disabled unit tests incompatible with Miri.
```

Comment:
Two problems with the changelog message:
* Integration tests are affected as well as unit tests.
* You disabled the tests conditionally, not unconditionally.
```suggestion
Fix running tests with Miri.
```
You should make a similar change to the commit message, too.